components/role-assignment/variables.tf

```
@@ -0,0 +1,20 @@
+variable "env" {
+  description = "Name of the environment to deploy the resource."
+}
+variable "product" {
+  description = "Name of the product."
+  default     = "shared-image-gallery"
+}
+
+variable "builtFrom" {
+  description = "Name of the GitHub repository this application is being built from."
+}
+
+variable "jenkins-mi" {
+  type = map(any)
+  default = {
+    "jenkins-cftptl-intsvc-mi"  = {},
```

can you use a list? you can call `toset` on it in the for each this syntax would then be a lot nicer.

```suggestion
    default = ["jenkins-cftptl-intsvc-mi", "jenkins-cftsbox-intsvc-mi", "jenkins-ptl-mi"]
```

---

Updated